src/App.css

```diff
@@ -379,5 +379,60 @@ Footer Styles
     font-weight: bold;
 }
 
+.navigationButton {
+    justify-content: flex-end;
+    background-color: white;
+}
+
+.legendContainer {
+    top: 5vh;
+    left:95%;
+    position: fixed;
+    z-index: 1;
+}
+
+.navContainer {
+    top: 25vh;
+    left:95%;
```

Comment:
For this one and the legend, when I try to click on mobile, I end up swiping the filter drawer open ~half the time. Could we move this further left?